### DISCUSSION TITLE
drum not aligned properly

### DISCUSSION THREAD

**Discussion by Jan-Arbus** [ppy/osu#21647] [Q&A]
Drums will not align properly if skin.ini is missing or there is no Version field in skin.ini
![image](https://user-images.githubusercontent.com/74862110/207874132-eb43e04e-b3fa-49ff-a814-2b0f16c65f00.png)

**Comment by peppy** [comment]
This is normal, and done this way to support an older version of skins which had different alignment. If this is not what you expect, could you please start by attaching your skin?

**Reply by Jan-Arbus** [reply]
so this is normal? Okay, I understand now, thanks.
